### PR TITLE
Edits

### PR DESCRIPTION
**Pull request by jpmedley**

### CODE REVIEW

File: gulp-tasks/_editAssist.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
+'use strict';
```
Comment (petele) [inline gulp-tasks/_editAssist.js]: Can you put this gulp task into a separate PR please?

File: src/content/en/updates/2016/04/devtools-digest-command-menu.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ description: Read about DevTools' new command menu and its over 60 actions that
 {# wf_tags: devtools,digest,update #}
 {# wf_featured_image: /web/updates/images/2016/04/devtools-digest/command-menu.png #}
 
-# DevTools Digest: More power with the new Command Menu {: .page-title }
+# DevTools Digest: More Power with the New Command Menu {: .page-title }
```
Comment (petele) [inline src/content/en/updates/2016/04/devtools-digest-command-menu.md]: Did title changes also get made in the _book.yaml and _toc.yaml?

File: src/content/en/updates/2016/04/keyboardevent-keys-codes.md [hunk 3]
```diff
@@ -15,30 +15,30 @@ description: Two new attributes bring consistent keyboard event handling to the
 
 The past few versions of Chrome have seen two additions to [`KeyboardEvent`s](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent), which are used as a parameter passed to [`keydown`](https://developer.mozilla.org/en-US/docs/Web/Events/keydown), [`keypress`](https://developer.mozilla.org/en-US/docs/Web/Events/keypress), and [`keyup`](https://developer.mozilla.org/en-US/docs/Web/Events/keyup) event listeners. Both the [`code`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code) attribute (added in [Chrome 48](https://www.chromestatus.com/feature/5228092293382144)) and the [`key`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key) attribute (added in [Chrome 51](https://www.chromestatus.com/feature/4748790720364544)) give developers a straightforward way to get information that would otherwise be [difficult](https://github.com/inexorabletash/polyfill/blob/master/keyboard.md#legacy-key-events) using legacy attributes.
 
-## The code Attribute
+## The code attribute
```
Comment (petele) [inline src/content/en/updates/2016/04/keyboardevent-keys-codes.md]: `code` should probably be wrapped in backticks since it's code

File: src/content/en/updates/2016/04/devtools-digest-command-menu.md (hunk 2 above)
Comment (jpmedley) [inline src/content/en/updates/2016/04/devtools-digest-command-menu.md]: For now they're manual.

File: src/content/en/updates/2016/04/keyboardevent-keys-codes.md (hunk 3 above)
Comment (jpmedley) [inline src/content/en/updates/2016/04/keyboardevent-keys-codes.md]: No, not in headings.

**Comment by jpmedley** [comment]
LGTM

File: src/content/en/updates/2016/04/devtools-digest-command-menu.md (hunk 2 above)
Comment (petele) [inline src/content/en/updates/2016/04/devtools-digest-command-menu.md]: For updates, shows, showcases they're automatically updated. But not for the static content
